Tests/LinuxMain.swift

```diff
@@ -1,70 +0,0 @@
-//===----------------------------------------------------------------------===//
```

Comment:
Can you also remove all of the `+XCTest.swift` under `Tests/`?

Reply:
7e1d8be